### PR TITLE
Browser compitable start button and other small revamps.

### PR DESCRIPTION
**Pull request by sudheerdarla**

### CODE REVIEW

**Comment by vramana** [comment]
Can you post Screenshot whenever you make visual changes?

**Comment by sudheerdarla** [comment]
fq
Here..Nothing much actually..just font and title.

**Comment by vramana** [comment]
Have you checked on other browsers as well?

**Comment by sudheerdarla** [comment]
Yes! It works fine as well in other browsers also.

File: style.css [hunk 1]
```diff
@@ -65,8 +60,8 @@ body {
     overflow: hidden;
     position: relative;
     margin : 0 auto;
-    top : 20px;
-    //margin-top : -50px;
+    top : 10px;
+    /*margin-top : -50px;*/
```
Comment (vramana) [inline style.css]: Can you also remove all the comments? They don't add anything.

File: style.css [hunk 2]
```diff
@@ -3,7 +3,6 @@ html, body {
     padding: 0;
     font-family: sans-serif;
 }
-
```
Comment (vramana) [inline style.css:5]: Don't remove white lines between declarations. They are good for readability. Can you add them back?

File: style.css [hunk 3]
```diff
@@ -13,40 +19,31 @@ body {
...
     padding: 0 10px;
     align-items: center;
 }
 
+.start {
+    background-color: red;
+    color : yellow;
+    width: 100px;
+    margin : 0 auto;
+    z-index: 9999;
```
Comment (vramana) [inline style.css]: Why 9999 ? Isn't 1 or 2 enough?

Comment (sudheerdarla) [reply style.css]: Yes! its enough! but I added like conventional way.

File: style.css (hunk 2 above)
Comment (duttakapil) [reply style.css:5]: You can edit the files on GitHub itself, right @vramana?

File: style.css (hunk 3 above)
Comment (vramana) [reply style.css]: Convention way is people have other reasons why they want to do it that way. We don't have any such reasons.

Comment (sudheerdarla) [reply style.css]: Okay I got it!

File: style.css [hunk 4]
```diff
@@ -13,40 +19,31 @@ body {
...
-}
-
-.start {
-    background-color: red;
-    color : yellow;
-    width: 100px;
-    margin : 0 auto;
+    position: absolute;
+    margin-top: -23%;
+    margin-left:46%;
```
Comment (vramana) [inline style.css:46]: Can you leave a space after `:` here ?

File: style.css [hunk 5]
```diff
@@ -65,8 +61,7 @@ body {
     overflow: hidden;
     position: relative;
     margin : 0 auto;
-    top : 20px;
-    //margin-top : -50px;
+    top : 10px;
```
Comment (vramana) [inline style.css:64]: Remove the space before `:`

Comment (vramana) [reply style.css:64]: It seems there are lot of other places where this happens can you correct them all?

File: style.css (hunk 4 above)
Comment (duttakapil) [reply style.css:46]: Nitpicks -_-

File: style.css (hunk 5 above)
Comment (sudheerdarla) [reply style.css:64]: I think those are not the major parts to be modified. but any way I'll do it..is there any thing you want me to modify.

Comment (duttakapil) [reply style.css:64]: Just let the guy have his first PR accepted already 😆

Comment (vramana) [reply style.css:64]: I know that. I just want things to get cleaned up while somebody is working on it rather push it to a different task.

Comment (vramana) [reply style.css:64]: @sudheerdarla Are you working on this or should I just merge? Either way is fine with me.

Comment (sudheerdarla) [reply style.css:64]: just merge! and lets move on to next version of the game of adding more features.

**Comment by vramana** [comment]
🎉 🎉

**Pull request merged**